cmd/cloud-run/rotate-signify-certificate/test_rotate_signify_certificate.py

```diff
@@ -0,0 +1,100 @@
+"""Module containing tests for signify secret rotator"""
+
+import base64
+import json
+from typing import Any, Dict
+import unittest
+from unittest.mock import MagicMock, Mock, patch
+from flask.testing import FlaskClient
+import test_fixtures
+from rotate_signify_certificate import app, get_pubsub_message, prepare_log_fields
+
+project_id: str = "test-project"
+component_name: str = "test_component"
+application_name: str = "test_application"
+
+
+class TestPubSubMessageHandler(unittest.TestCase):
+    """Test cases for fucntion to handle pub sub message"""
+
+    def setUp(self) -> None:
+        self.app: FlaskClient = app.test_client()
+        self.app.testing = True
+
+    @patch("rotate_signify_certificate.project_id", project_id)
+    @patch("rotate_signify_certificate.component_name", component_name)
+    @patch("rotate_signify_certificate.application_name", application_name)
+    def test_prepare_log_fields(self):
+        """Test that prepare log fields based on current env variables and cloud trace context"""
+        with app.test_request_context(
+            "/", headers={"X-Cloud-Trace-Context": "trace-id/other-info"}
+        ):
+            log_fields: Dict[str, Any] = prepare_log_fields()
+            self.assertIn("logging.googleapis.com/trace", log_fields)
+            self.assertEqual(
+                log_fields["logging.googleapis.com/trace"],
+                f"projects/{project_id}/traces/trace-id",
+            )
+            self.assertEqual(log_fields["Component"], "signify-certificate-rotator")
```

Comment:
```suggestion
            self.assertEqual(log_fields["component"], "signify-certificate-rotator")
```

Reply:
I created the issue to share that code, I prefer to keep it similar to slack-message-sender one: https://github.com/orgs/kyma-project/projects/22/views/51?pane=issue&itemId=79537174